packages/web/src/components/common/pool-graph/pool-graph-svg/PoolGraphSVG.tsx

```diff
@@ -86,6 +86,8 @@ const PoolGraphSVG = forwardRef<SVGSVGElement, PoolGraphSVGProps>(
     const { defaultMinX, minX, maxX } = d3Position;
     const centerX = currentTick ?? (minX && maxX ? (minX + maxX) / 2 : 0);
 
+    const hasCurrentTick = React.useMemo(() => currentTick !== null && currentTick !== undefined, [currentTick]);
```

Comment:
niptick: `!= null` checks that the value is neither null nor undefined

```suggestion
    const hasCurrentTick = React.useMemo(() => currentTick != null, [currentTick]);
```